Index: sfd/Makefile
===================================================================
--- sfd/Makefile	(revision 4273)
+++ sfd/Makefile	(working copy)
@@ -45,8 +45,8 @@
 %.otf : %.sfd
 	@ $(TESTFF)
 	@ ( $(FFPY) ${GENOT} $< 3>&1 1>&2 2>&3 | tail -n +4 ) 3>&1 1>&2 2>&3 2>&1 
-	@ ( $(IFP) $@ 3>&1 1>&2 2>&3 | tail -n +4 ) 
-	@ ( $(CFS) $@ 3>&1 1>&2 2>&3 | tail -n +4 ) 
+	#@ ( $(IFP) $@ 3>&1 1>&2 2>&3 | tail -n +4 ) 
+	#@ ( $(CFS) $@ 3>&1 1>&2 2>&3 | tail -n +4 ) 
 
 %.ttf : %.sfd
 	@ $(TESTFF)
Index: tools/generate/OpenType
===================================================================
--- tools/generate/OpenType	(revision 4273)
+++ tools/generate/OpenType	(working copy)
@@ -45,7 +45,7 @@
 	# but all attemtps to determine that from Python failed.
 	f.encoding = 'UnicodeFull'
 	f.selection.all()
-	f.autoHint()	# because without gasp table, small sizes ugly
+	#f.autoHint()	# because without gasp table, small sizes ugly
 	# 'old-kern' and 'dummy-dsig' are work-arounds for Windows
 	f.generate( otfile, flags=('opentype','round') )
 	f.close()
